### PR TITLE
Current measurement

### PR DESCRIPTION
**Pull request by deJcob**
Added current measurement

### CODE REVIEW

File: PBLV5/Core/Src/CurrentMeasurement.cpp [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	lastNDTR = dmaHandler->Instance->NDTR;
+}
+
+
+uint16_t CurrentMeasurement::getCurrentData(uint8_t *dataBuffer)
+{
+	uint32_t currentNDTR = dmaHandler->Instance->NDTR;
+	uint32_t len = 0;
+
+	if (currentNDTR % 2 != 0) currentNDTR++; // I want to have got data pairs and also do not go outside the memory
```
Comment (JacekJurek) [inline PBLV5/Core/Src/CurrentMeasurement.cpp]: Consider making function that will have name according to comment that is here so the comment will be redundant.

File: PBLV5/Core/Src/CurrentMeasurement.cpp [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	{
+		len = lastNDTR - currentNDTR;
+		std::copy(dataBuff+(currentNDTR*MEASUREMENT_BYTES), dataBuff+(lastNDTR*MEASUREMENT_BYTES), dataBuffer);
+	}
+	else
+	{
+		len = lastNDTR;
+		std::copy(dataBuff, dataBuff+(lastNDTR*MEASUREMENT_BYTES), dataBuffer);
+		len += ADC_BUF_LEN - currentNDTR;
+		std::copy(dataBuff+(currentNDTR*MEASUREMENT_BYTES), dataBuff+ADC_BUF_LEN_IN_BYTES, dataBuffer+(lastNDTR*MEASUREMENT_BYTES));
```
Comment (JacekJurek) [inline PBLV5/Core/Src/CurrentMeasurement.cpp]: Consider separating some operations with properly named functions to give reviewer a hint what is happening inside `if/else` braces.

File: PBLV5/Core/Src/CurrentMeasurement.cpp (hunk 1 above)
Comment (deJcob) [reply PBLV5/Core/Src/CurrentMeasurement.cpp]: Changed in last commit, please check it @JacekJurek

File: PBLV5/Core/Src/CurrentMeasurement.cpp (hunk 2 above)
Comment (deJcob) [reply PBLV5/Core/Src/CurrentMeasurement.cpp]: Changed in last commit, please check it @JacekJurek

File: PBLV5/Core/Src/CurrentMeasurement.cpp (hunk 1 above)
Comment (JacekJurek) [reply PBLV5/Core/Src/CurrentMeasurement.cpp]: Looks good now

File: PBLV5/Core/Src/CurrentMeasurement.cpp (hunk 2 above)
Comment (JacekJurek) [reply PBLV5/Core/Src/CurrentMeasurement.cpp]: Good.

**Comment by JacekJurek** [comment]
Can be merged when changes associated with currentMeasurement will be ready to merge on `vc200` project

File: PBLV5/Core/Src/main.cpp [hunk 3]
```diff
@@ -437,6 +439,7 @@ int main(void)
   imuSensors.initializeI2C_Sensors(&hi2c1);
   rulerSensors.initializeI2C_Sensors(&hi2c2);
 
+  currentMeasurement.Init();
```
Comment (JacekJurek) [inline PBLV5/Core/Src/main.cpp]: please start function names from small letter

File: PBLV5/PBLV5 Debug Linux.launch [hunk 4]
```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (JacekJurek) [inline PBLV5/PBLV5 Debug Linux.launch]: please revert and don't commit such files to repository

**Pull request merged**